tests/tests/test_monitor_client.py

```diff
@@ -215,6 +246,41 @@ def test_monitorclient_alert_email(self, monitor_commercial_setup_no_client):
         )
         logger.info("test_monitorclient_alert_email: got OK alert email.")
 
+        logger.info(
+            "test_monitorclient_alert_email: email alert on log file containing a pattern scenario."
+        )
+        log_file = "/tmp/mylog.log"
+        mender_device.run("echo 'some line' >> " + log_file)
+        prepare_log_monitoring(
+            mender_device, service_name, log_file, "session opened for user [a-z]*",
+        )
+        time.sleep(2 * wait_for_alert_interval_s)
+        mender_device.run("echo 'some line' >> " + log_file)
+        time.sleep(wait_for_alert_interval_s)
```

Comment:
Around here we should `assert` that no e-mail has been received (iow the pattern has not yet been found)

Reply:
added!